qcelemental/_version.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Better to put in a yapf disable around whole file than let a versioneer generates file be edited, right?

Reply:
I would say, yes. But since we do not regenerate these, I say just YAPF once and get on with it.

Reply:
So long as doesn’t interfere with upgrading versioneer, sounds fine. (I don’t think it should, just the one complication that comes to mind)

Reply:
I don't think versioneer will even be upgraded again to be honest. I think that repository is dead and PyPA is creating a replacement. See [here](https://github.com/pypa/setuptools_scm).